plugins/sqlfluff-plugin-example/src/example/rules.py

```diff
@@ -63,15 +63,17 @@ class Rule_Example_L001(BaseCrawler):
         ORDER BY bar
     """
 
-    # Binary operators behave like keywords too.
-    _target_elems: List[Tuple[str, str]] = [("type", "orderby_clause")]
     config_keywords = ["forbidden_columns"]
 
+    def __init__(self, code, description, **kwargs):
```

Comment:
Nitpick: Since the class-specific `__init__()` function isn't using `code` or `description`, you could do this instead:
```
def __init__(self, *args, **kwargs):
    super().__init__(*args, **kwargs)
```

No big deal, just a bit less code, and avoids having to change this code if the base class parameter list changes.

Reply:
Ah makes sense! I changed that and the other nitpick